crates/sqlbuiltins/src/functions/object_store.rs

```diff
@@ -39,6 +41,16 @@ impl TableFunc for ObjScanTableFunc {
         RuntimePreference::Unspecified
     }
 
+    fn signature(&self) -> Option<Signature> {
+        Some(Signature::uniform(
+            1,
+            vec![
+                DataType::Utf8,
+                DataType::List(Arc::new(Field::new("url", DataType::Utf8, false))),
```

Comment:
```suggestion
                DataType::List(Arc::new(Field::new("item", DataType::Utf8, false))),
```

"item" should be used when specifying data types for lists.